src/rosie.js

```diff
@@ -399,6 +399,17 @@ Factory.util = (function() {
         }
       }
       return dest;
+    },
+
+    /**
+     * Deep clones given object.
+     *
+     * @private
+     * @param {object} object
+     * @return {object}
+     */
+    deepClone: function(object) {
+      return JSON.parse(JSON.stringify(object));
```

Comment:
This would be a breaking change in that it wouldn't support using anything that doesn't serialize as JSON as attribute values.